pytest/tests/mocknet/helpers/load_test_spoon_helper.py

```diff
@@ -75,14 +97,16 @@ def get_test_accounts_from_args(argv):
     )
 
 
-def main(argv):
-    logger.info(argv)
-    test_state = get_test_accounts_from_args(argv)
+def main():
+    logger.info(" ".join(sys.argv))
+
+    args = parse_args()
+    test_state = get_test_accounts_from_args(args)
 
     # Ensure load testing contract is deployed to all accounts before
     # starting to send random transactions (ensures we do not try to
     # call the contract before it is deployed).
-    delay = CONTRACT_DEPLOY_TIME / test_state.num_test_accounts()
+    delay = args.contract_deploy_time / test_state.num_test_accounts()
```

Comment:
I remember getting the errors that the transaction can't be submitted. Worth giving it another try.

Reply:
If this is still reproducible, it sounds to me like a potentially serious issue to look into. Unless we're just overloading the network with deployments and the transactions are discarded, of course.

Reply:
@nagisa Can you point me to where transactions can be discarded? Do we have any other rate-limiting or similar mechanism in place to prevent overloading the node?

Reply:
I’m just retelling hearsay so I don’t have a good pointer here, sorry.